log/handler_glog.go

```diff
@@ -192,7 +192,7 @@ func (h *GlogHandler) Handle(_ context.Context, r slog.Record) error {
 		frame, _ := fs.Next()
 
 		for _, rule := range h.patterns {
-			if rule.pattern.MatchString(fmt.Sprintf("%+s", frame.File)) {
+			if rule.pattern.MatchString(fmt.Sprintf("%s", frame.File)) {
```

Comment:
Why did this change?

Reply:
Well, you are matching a string against some regexp. Why change it if there's no need for it? 
Please revert